allennlp/training/metrics/fbeta_multi_label_measure.py

```diff
@@ -0,0 +1,210 @@
+from typing import List, Optional, Union
+
+import torch
+import torch.distributed as dist
+from overrides import overrides
+
+from allennlp.common.checks import ConfigurationError
+from allennlp.common.util import is_distributed
+from allennlp.training.metrics import FBetaMeasure
+from allennlp.training.metrics.metric import Metric
+
+
+@Metric.register("fbeta_multi_label")
+class FBetaMultiLabelMeasure(FBetaMeasure):
```

Comment:
It might be good to have a multi-label F1 that is just this class with a hard-coded `1.0` for beta. People will be looking for that.